lumicks/pylake/force_calibration/power_spectrum_calibration.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, model, ps_model_fit, ps_fitted, params, results):
         self.results = results
 
         # A few parameters have to be present for this calibration to be used.
-        mandatory_params = ["kappa (pN/nm)", "Rd (um/V)", "Rf (pN/V)"]
+        mandatory_params = ["kappa (pN/nm)", "Rf (pN/V)"]
```

Comment:
I don't think it's obvious why this was removed. It would be good to include an explanation in the commit description: "It's not mandatory because the value can be calculated from the other two mandatory fields".